roachpb: TestMakePriority failed

roachpb.TestMakePriority [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/11703437?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/11703437?buildTab=artifacts#/) on master @ [46fedddafbff155b72e3853c4f32157cbfebcd35](https://github.com/cockroachdb/cockroach/commits/46fedddafbff155b72e3853c4f32157cbfebcd35):


```
=== RUN   TestMakePriority
    data_test.go:976: 0.001100 vs 0.010000: 9738 wins, 90262 losses (winRatio=0.107886, priRatio=0.110000)
    data_test.go:976: 0.001100 vs 0.100000: 1108 wins, 98892 losses (winRatio=0.011204, priRatio=0.011000)
    data_test.go:976: 0.001100 vs 0.500000: 228 wins, 99772 losses (winRatio=0.002285, priRatio=0.002200)
    data_test.go:976: 0.001100 vs 1.000000: 132 wins, 99868 losses (winRatio=0.001322, priRatio=0.001100)
    data_test.go:979: Error (0.001100 vs 1.000000: 132 wins, 99868 losses): winRatio=0.001322 not close to priRatio=0.001100
    data_test.go:976: 0.001100 vs 1.000000: 111 wins, 99889 losses (winRatio=0.001111, priRatio=0.001100)
    data_test.go:976: 0.010000 vs 0.100000: 9126 wins, 90874 losses (winRatio=0.100425, priRatio=0.100000)
    data_test.go:976: 0.010000 vs 0.500000: 1987 wins, 98013 losses (winRatio=0.020273, priRatio=0.020000)
    data_test.go:976: 0.010000 vs 1.000000: 951 wins, 99049 losses (winRatio=0.009601, priRatio=0.010000)
    data_test.go:976: 0.010000 vs 1.000000: 948 wins, 99052 losses (winRatio=0.009571, priRatio=0.010000)
    data_test.go:976: 0.010000 vs 2.000000: 500 wins, 99500 losses (winRatio=0.005025, priRatio=0.005000)
    data_test.go:976: 0.100000 vs 0.500000: 16869 wins, 83131 losses (winRatio=0.202921, priRatio=0.200000)
    data_test.go:976: 0.100000 vs 1.000000: 9099 wins, 90901 losses (winRatio=0.100098, priRatio=0.100000)
    data_test.go:976: 0.100000 vs 1.000000: 9147 wins, 90853 losses (winRatio=0.100679, priRatio=0.100000)
    data_test.go:976: 0.100000 vs 2.000000: 4781 wins, 95219 losses (winRatio=0.050211, priRatio=0.050000)
    data_test.go:976: 0.100000 vs 10.000000: 972 wins, 99028 losses (winRatio=0.009815, priRatio=0.010000)
    data_test.go:976: 0.500000 vs 1.000000: 33105 wins, 66895 losses (winRatio=0.494880, priRatio=0.500000)
    data_test.go:976: 0.500000 vs 1.000000: 33231 wins, 66769 losses (winRatio=0.497701, priRatio=0.500000)
    data_test.go:976: 0.500000 vs 2.000000: 19814 wins, 80186 losses (winRatio=0.247100, priRatio=0.250000)
    data_test.go:976: 0.500000 vs 10.000000: 4740 wins, 95260 losses (winRatio=0.049759, priRatio=0.050000)
    data_test.go:976: 0.500000 vs 100.000000: 521 wins, 99479 losses (winRatio=0.005237, priRatio=0.005000)
    data_test.go:976: 1.000000 vs 1.000000: 50024 wins, 49976 losses (winRatio=1.000960, priRatio=1.000000)
    data_test.go:976: 1.000000 vs 2.000000: 33376 wins, 66624 losses (winRatio=0.500961, priRatio=0.500000)
    data_test.go:976: 1.000000 vs 10.000000: 9175 wins, 90825 losses (winRatio=0.101018, priRatio=0.100000)
    data_test.go:976: 1.000000 vs 100.000000: 998 wins, 99002 losses (winRatio=0.010081, priRatio=0.010000)
    data_test.go:976: 1.000000 vs 999.000000: 104 wins, 99896 losses (winRatio=0.001041, priRatio=0.001001)
    data_test.go:976: 1.000000 vs 2.000000: 33338 wins, 66662 losses (winRatio=0.500105, priRatio=0.500000)
    data_test.go:976: 1.000000 vs 10.000000: 9126 wins, 90874 losses (winRatio=0.100425, priRatio=0.100000)
    data_test.go:976: 1.000000 vs 100.000000: 1004 wins, 98996 losses (winRatio=0.010142, priRatio=0.010000)
    data_test.go:976: 1.000000 vs 999.000000: 107 wins, 99893 losses (winRatio=0.001071, priRatio=0.001001)
    data_test.go:976: 2.000000 vs 10.000000: 16520 wins, 83480 losses (winRatio=0.197892, priRatio=0.200000)
    data_test.go:976: 2.000000 vs 100.000000: 1959 wins, 98041 losses (winRatio=0.019981, priRatio=0.020000)
    data_test.go:976: 2.000000 vs 999.000000: 216 wins, 99784 losses (winRatio=0.002165, priRatio=0.002002)
    data_test.go:976: 10.000000 vs 100.000000: 9069 wins, 90931 losses (winRatio=0.099735, priRatio=0.100000)
    data_test.go:976: 10.000000 vs 999.000000: 1054 wins, 98946 losses (winRatio=0.010652, priRatio=0.010010)
    data_test.go:976: 100.000000 vs 999.000000: 9098 wins, 90902 losses (winRatio=0.100086, priRatio=0.100100)
--- FAIL: TestMakePriority (0.03s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
, <code>stress=true</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/kv
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMakePriority.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-31364

roachpb.TestMakePriority [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/11703435?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/11703435?buildTab=artifacts#/) on master @ [46fedddafbff155b72e3853c4f32157cbfebcd35](https://github.com/cockroachdb/cockroach/commits/46fedddafbff155b72e3853c4f32157cbfebcd35):


```
=== RUN   TestMakePriority
    data_test.go:976: 0.001100 vs 0.010000: 9799 wins, 90201 losses (winRatio=0.108635, priRatio=0.110000)
    data_test.go:976: 0.001100 vs 0.100000: 1076 wins, 98924 losses (winRatio=0.010877, priRatio=0.011000)
    data_test.go:976: 0.001100 vs 0.500000: 198 wins, 99802 losses (winRatio=0.001984, priRatio=0.002200)
    data_test.go:976: 0.001100 vs 1.000000: 87 wins, 99913 losses (winRatio=0.000871, priRatio=0.001100)
    data_test.go:979: Error (0.001100 vs 1.000000: 87 wins, 99913 losses): winRatio=0.000871 not close to priRatio=0.001100
    data_test.go:976: 0.001100 vs 1.000000: 125 wins, 99875 losses (winRatio=0.001252, priRatio=0.001100)
    data_test.go:976: 0.010000 vs 0.100000: 9226 wins, 90774 losses (winRatio=0.101637, priRatio=0.100000)
    data_test.go:976: 0.010000 vs 0.500000: 1954 wins, 98046 losses (winRatio=0.019929, priRatio=0.020000)
    data_test.go:976: 0.010000 vs 1.000000: 1001 wins, 98999 losses (winRatio=0.010111, priRatio=0.010000)
    data_test.go:976: 0.010000 vs 1.000000: 983 wins, 99017 losses (winRatio=0.009928, priRatio=0.010000)
    data_test.go:976: 0.010000 vs 2.000000: 529 wins, 99471 losses (winRatio=0.005318, priRatio=0.005000)
    data_test.go:976: 0.100000 vs 0.500000: 16653 wins, 83347 losses (winRatio=0.199803, priRatio=0.200000)
    data_test.go:976: 0.100000 vs 1.000000: 9037 wins, 90963 losses (winRatio=0.099348, priRatio=0.100000)
    data_test.go:976: 0.100000 vs 1.000000: 9146 wins, 90854 losses (winRatio=0.100667, priRatio=0.100000)
    data_test.go:976: 0.100000 vs 2.000000: 4794 wins, 95206 losses (winRatio=0.050354, priRatio=0.050000)
    data_test.go:976: 0.100000 vs 10.000000: 997 wins, 99003 losses (winRatio=0.010070, priRatio=0.010000)
    data_test.go:976: 0.500000 vs 1.000000: 33404 wins, 66596 losses (winRatio=0.501592, priRatio=0.500000)
    data_test.go:976: 0.500000 vs 1.000000: 33294 wins, 66706 losses (winRatio=0.499116, priRatio=0.500000)
    data_test.go:976: 0.500000 vs 2.000000: 19928 wins, 80072 losses (winRatio=0.248876, priRatio=0.250000)
    data_test.go:976: 0.500000 vs 10.000000: 4752 wins, 95248 losses (winRatio=0.049891, priRatio=0.050000)
    data_test.go:976: 0.500000 vs 100.000000: 510 wins, 99490 losses (winRatio=0.005126, priRatio=0.005000)
    data_test.go:976: 1.000000 vs 1.000000: 49953 wins, 50047 losses (winRatio=0.998122, priRatio=1.000000)
    data_test.go:976: 1.000000 vs 2.000000: 33587 wins, 66413 losses (winRatio=0.505729, priRatio=0.500000)
    data_test.go:976: 1.000000 vs 10.000000: 9066 wins, 90934 losses (winRatio=0.099699, priRatio=0.100000)
    data_test.go:976: 1.000000 vs 100.000000: 1000 wins, 99000 losses (winRatio=0.010101, priRatio=0.010000)
    data_test.go:976: 1.000000 vs 999.000000: 88 wins, 99912 losses (winRatio=0.000881, priRatio=0.001001)
    data_test.go:976: 1.000000 vs 2.000000: 33429 wins, 66571 losses (winRatio=0.502156, priRatio=0.500000)
    data_test.go:976: 1.000000 vs 10.000000: 9071 wins, 90929 losses (winRatio=0.099759, priRatio=0.100000)
    data_test.go:976: 1.000000 vs 100.000000: 970 wins, 99030 losses (winRatio=0.009795, priRatio=0.010000)
    data_test.go:976: 1.000000 vs 999.000000: 87 wins, 99913 losses (winRatio=0.000871, priRatio=0.001001)
    data_test.go:976: 2.000000 vs 10.000000: 16598 wins, 83402 losses (winRatio=0.199012, priRatio=0.200000)
    data_test.go:976: 2.000000 vs 100.000000: 1924 wins, 98076 losses (winRatio=0.019617, priRatio=0.020000)
    data_test.go:976: 2.000000 vs 999.000000: 205 wins, 99795 losses (winRatio=0.002054, priRatio=0.002002)
    data_test.go:976: 10.000000 vs 100.000000: 9024 wins, 90976 losses (winRatio=0.099191, priRatio=0.100000)
    data_test.go:976: 10.000000 vs 999.000000: 992 wins, 99008 losses (winRatio=0.010019, priRatio=0.010010)
    data_test.go:976: 100.000000 vs 999.000000: 9081 wins, 90919 losses (winRatio=0.099880, priRatio=0.100100)
--- FAIL: TestMakePriority (0.03s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
, <code>stress=true</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMakePriority.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachpb.TestMakePriority [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/11712166?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/11712166?buildTab=artifacts#/) on master @ [b5836a5b4ab6c4fe58ada6002eb8c56c4e35438e](https://github.com/cockroachdb/cockroach/commits/b5836a5b4ab6c4fe58ada6002eb8c56c4e35438e):


```
=== RUN   TestMakePriority
    data_test.go:976: 0.001100 vs 0.010000: 9900 wins, 90100 losses (winRatio=0.109878, priRatio=0.110000)
    data_test.go:976: 0.001100 vs 0.100000: 1101 wins, 98899 losses (winRatio=0.011133, priRatio=0.011000)
    data_test.go:976: 0.001100 vs 0.500000: 208 wins, 99792 losses (winRatio=0.002084, priRatio=0.002200)
    data_test.go:976: 0.001100 vs 1.000000: 136 wins, 99864 losses (winRatio=0.001362, priRatio=0.001100)
    data_test.go:979: Error (0.001100 vs 1.000000: 136 wins, 99864 losses): winRatio=0.001362 not close to priRatio=0.001100
    data_test.go:976: 0.001100 vs 1.000000: 88 wins, 99912 losses (winRatio=0.000881, priRatio=0.001100)
    data_test.go:976: 0.010000 vs 0.100000: 9113 wins, 90887 losses (winRatio=0.100267, priRatio=0.100000)
    data_test.go:976: 0.010000 vs 0.500000: 1917 wins, 98083 losses (winRatio=0.019545, priRatio=0.020000)
    data_test.go:976: 0.010000 vs 1.000000: 997 wins, 99003 losses (winRatio=0.010070, priRatio=0.010000)
    data_test.go:976: 0.010000 vs 1.000000: 972 wins, 99028 losses (winRatio=0.009815, priRatio=0.010000)
    data_test.go:976: 0.010000 vs 2.000000: 505 wins, 99495 losses (winRatio=0.005076, priRatio=0.005000)
    data_test.go:976: 0.100000 vs 0.500000: 16588 wins, 83412 losses (winRatio=0.198868, priRatio=0.200000)
    data_test.go:976: 0.100000 vs 1.000000: 9010 wins, 90990 losses (winRatio=0.099022, priRatio=0.100000)
    data_test.go:976: 0.100000 vs 1.000000: 9077 wins, 90923 losses (winRatio=0.099832, priRatio=0.100000)
    data_test.go:976: 0.100000 vs 2.000000: 4861 wins, 95139 losses (winRatio=0.051094, priRatio=0.050000)
    data_test.go:976: 0.100000 vs 10.000000: 995 wins, 99005 losses (winRatio=0.010050, priRatio=0.010000)
    data_test.go:976: 0.500000 vs 1.000000: 33283 wins, 66717 losses (winRatio=0.498868, priRatio=0.500000)
    data_test.go:976: 0.500000 vs 1.000000: 33541 wins, 66459 losses (winRatio=0.504687, priRatio=0.500000)
    data_test.go:976: 0.500000 vs 2.000000: 20126 wins, 79874 losses (winRatio=0.251972, priRatio=0.250000)
    data_test.go:976: 0.500000 vs 10.000000: 4819 wins, 95181 losses (winRatio=0.050630, priRatio=0.050000)
    data_test.go:976: 0.500000 vs 100.000000: 497 wins, 99503 losses (winRatio=0.004995, priRatio=0.005000)
    data_test.go:976: 1.000000 vs 1.000000: 50233 wins, 49767 losses (winRatio=1.009364, priRatio=1.000000)
    data_test.go:976: 1.000000 vs 2.000000: 33554 wins, 66446 losses (winRatio=0.504981, priRatio=0.500000)
    data_test.go:976: 1.000000 vs 10.000000: 9176 wins, 90824 losses (winRatio=0.101031, priRatio=0.100000)
    data_test.go:976: 1.000000 vs 100.000000: 995 wins, 99005 losses (winRatio=0.010050, priRatio=0.010000)
    data_test.go:976: 1.000000 vs 999.000000: 92 wins, 99908 losses (winRatio=0.000921, priRatio=0.001001)
    data_test.go:976: 1.000000 vs 2.000000: 33680 wins, 66320 losses (winRatio=0.507841, priRatio=0.500000)
    data_test.go:976: 1.000000 vs 10.000000: 9163 wins, 90837 losses (winRatio=0.100873, priRatio=0.100000)
    data_test.go:976: 1.000000 vs 100.000000: 972 wins, 99028 losses (winRatio=0.009815, priRatio=0.010000)
    data_test.go:976: 1.000000 vs 999.000000: 99 wins, 99901 losses (winRatio=0.000991, priRatio=0.001001)
    data_test.go:976: 2.000000 vs 10.000000: 16779 wins, 83221 losses (winRatio=0.201620, priRatio=0.200000)
    data_test.go:976: 2.000000 vs 100.000000: 1951 wins, 98049 losses (winRatio=0.019898, priRatio=0.020000)
    data_test.go:976: 2.000000 vs 999.000000: 204 wins, 99796 losses (winRatio=0.002044, priRatio=0.002002)
    data_test.go:976: 10.000000 vs 100.000000: 9021 wins, 90979 losses (winRatio=0.099155, priRatio=0.100000)
    data_test.go:976: 10.000000 vs 999.000000: 1002 wins, 98998 losses (winRatio=0.010121, priRatio=0.010010)
    data_test.go:976: 100.000000 vs 999.000000: 9139 wins, 90861 losses (winRatio=0.100582, priRatio=0.100100)
--- FAIL: TestMakePriority (0.04s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
, <code>stress=true</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMakePriority.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachpb.TestMakePriority [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/11712164?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/11712164?buildTab=artifacts#/) on master @ [b5836a5b4ab6c4fe58ada6002eb8c56c4e35438e](https://github.com/cockroachdb/cockroach/commits/b5836a5b4ab6c4fe58ada6002eb8c56c4e35438e):


```
=== RUN   TestMakePriority
F230910 11:35:21.394742 86 roachpb/data.go:1228  [-] 3  updating txn meta={id=b5157a47 key=/Min iso=Serializable pri=0.00000000 epo=0 ts=0,0 min=0,0 seq=0} lock=false stat=PENDING rts=0,0 wto=false gul=0,0 with different txn "name" meta={id=24eb9256 key="foo" iso=Snapshot pri=44.58039917 epo=2 ts=0.000000020,21? min=0.000000010,11? seq=123} lock=true stat=COMMITTED rts=0.000000020,22? wto=true gul=0.000000040,41? int=1 ifw=1 isn=1
F230910 11:35:21.394742 86 roachpb/data.go:1228  [-] 3 !stack trace omitted via SetExitFunc()
    data_test.go:976: 0.001100 vs 0.010000: 9860 wins, 90140 losses (winRatio=0.109385, priRatio=0.110000)
    data_test.go:976: 0.001100 vs 0.100000: 1097 wins, 98903 losses (winRatio=0.011092, priRatio=0.011000)
    data_test.go:976: 0.001100 vs 0.500000: 184 wins, 99816 losses (winRatio=0.001843, priRatio=0.002200)
    data_test.go:976: 0.001100 vs 1.000000: 125 wins, 99875 losses (winRatio=0.001252, priRatio=0.001100)
    data_test.go:976: 0.001100 vs 1.000000: 108 wins, 99892 losses (winRatio=0.001081, priRatio=0.001100)
    data_test.go:976: 0.010000 vs 0.100000: 9155 wins, 90845 losses (winRatio=0.100776, priRatio=0.100000)
    data_test.go:976: 0.010000 vs 0.500000: 1891 wins, 98109 losses (winRatio=0.019274, priRatio=0.020000)
    data_test.go:976: 0.010000 vs 1.000000: 1056 wins, 98944 losses (winRatio=0.010673, priRatio=0.010000)
    data_test.go:976: 0.010000 vs 1.000000: 978 wins, 99022 losses (winRatio=0.009877, priRatio=0.010000)
    data_test.go:976: 0.010000 vs 2.000000: 504 wins, 99496 losses (winRatio=0.005066, priRatio=0.005000)
    data_test.go:976: 0.100000 vs 0.500000: 16782 wins, 83218 losses (winRatio=0.201663, priRatio=0.200000)
    data_test.go:976: 0.100000 vs 1.000000: 9177 wins, 90823 losses (winRatio=0.101043, priRatio=0.100000)
    data_test.go:976: 0.100000 vs 1.000000: 8924 wins, 91076 losses (winRatio=0.097984, priRatio=0.100000)
    data_test.go:976: 0.100000 vs 2.000000: 4699 wins, 95301 losses (winRatio=0.049307, priRatio=0.050000)
    data_test.go:976: 0.100000 vs 10.000000: 973 wins, 99027 losses (winRatio=0.009826, priRatio=0.010000)
    data_test.go:976: 0.500000 vs 1.000000: 33194 wins, 66806 losses (winRatio=0.496872, priRatio=0.500000)
    data_test.go:976: 0.500000 vs 1.000000: 33091 wins, 66909 losses (winRatio=0.494567, priRatio=0.500000)
    data_test.go:976: 0.500000 vs 2.000000: 19950 wins, 80050 losses (winRatio=0.249219, priRatio=0.250000)
    data_test.go:976: 0.500000 vs 10.000000: 4777 wins, 95223 losses (winRatio=0.050166, priRatio=0.050000)
    data_test.go:976: 0.500000 vs 100.000000: 531 wins, 99469 losses (winRatio=0.005338, priRatio=0.005000)
    data_test.go:976: 1.000000 vs 1.000000: 49893 wins, 50107 losses (winRatio=0.995729, priRatio=1.000000)
    data_test.go:976: 1.000000 vs 2.000000: 33288 wins, 66712 losses (winRatio=0.498981, priRatio=0.500000)
    data_test.go:976: 1.000000 vs 10.000000: 9220 wins, 90780 losses (winRatio=0.101564, priRatio=0.100000)
    data_test.go:976: 1.000000 vs 100.000000: 1007 wins, 98993 losses (winRatio=0.010172, priRatio=0.010000)
    data_test.go:976: 1.000000 vs 999.000000: 123 wins, 99877 losses (winRatio=0.001232, priRatio=0.001001)
    data_test.go:979: Error (1.000000 vs 999.000000: 123 wins, 99877 losses): winRatio=0.001232 not close to priRatio=0.001001
    data_test.go:976: 1.000000 vs 2.000000: 33330 wins, 66670 losses (winRatio=0.499925, priRatio=0.500000)
    data_test.go:976: 1.000000 vs 10.000000: 9050 wins, 90950 losses (winRatio=0.099505, priRatio=0.100000)
    data_test.go:976: 1.000000 vs 100.000000: 1001 wins, 98999 losses (winRatio=0.010111, priRatio=0.010000)
    data_test.go:976: 1.000000 vs 999.000000: 110 wins, 99890 losses (winRatio=0.001101, priRatio=0.001001)
    data_test.go:976: 2.000000 vs 10.000000: 16678 wins, 83322 losses (winRatio=0.200163, priRatio=0.200000)
    data_test.go:976: 2.000000 vs 100.000000: 1950 wins, 98050 losses (winRatio=0.019888, priRatio=0.020000)
    data_test.go:976: 2.000000 vs 999.000000: 233 wins, 99767 losses (winRatio=0.002335, priRatio=0.002002)
    data_test.go:976: 10.000000 vs 100.000000: 9165 wins, 90835 losses (winRatio=0.100897, priRatio=0.100000)
    data_test.go:976: 10.000000 vs 999.000000: 1038 wins, 98962 losses (winRatio=0.010489, priRatio=0.010010)
    data_test.go:976: 100.000000 vs 999.000000: 9153 wins, 90847 losses (winRatio=0.100752, priRatio=0.100100)
--- FAIL: TestMakePriority (0.04s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
, <code>stress=true</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMakePriority.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachpb.TestMakePriority [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/11720720?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/11720720?buildTab=artifacts#/) on master @ [9959523f444ac2e764d6b2724df35368bc4d0d8a](https://github.com/cockroachdb/cockroach/commits/9959523f444ac2e764d6b2724df35368bc4d0d8a):


```
=== RUN   TestMakePriority
    data_test.go:976: 0.001100 vs 0.010000: 9986 wins, 90014 losses (winRatio=0.110938, priRatio=0.110000)
    data_test.go:976: 0.001100 vs 0.100000: 1122 wins, 98878 losses (winRatio=0.011347, priRatio=0.011000)
    data_test.go:976: 0.001100 vs 0.500000: 199 wins, 99801 losses (winRatio=0.001994, priRatio=0.002200)
    data_test.go:976: 0.001100 vs 1.000000: 111 wins, 99889 losses (winRatio=0.001111, priRatio=0.001100)
    data_test.go:976: 0.001100 vs 1.000000: 114 wins, 99886 losses (winRatio=0.001141, priRatio=0.001100)
    data_test.go:976: 0.010000 vs 0.100000: 9263 wins, 90737 losses (winRatio=0.102086, priRatio=0.100000)
    data_test.go:976: 0.010000 vs 0.500000: 1975 wins, 98025 losses (winRatio=0.020148, priRatio=0.020000)
    data_test.go:976: 0.010000 vs 1.000000: 952 wins, 99048 losses (winRatio=0.009612, priRatio=0.010000)
    data_test.go:976: 0.010000 vs 1.000000: 1015 wins, 98985 losses (winRatio=0.010254, priRatio=0.010000)
    data_test.go:976: 0.010000 vs 2.000000: 459 wins, 99541 losses (winRatio=0.004611, priRatio=0.005000)
    data_test.go:976: 0.100000 vs 0.500000: 16816 wins, 83184 losses (winRatio=0.202154, priRatio=0.200000)
    data_test.go:976: 0.100000 vs 1.000000: 9138 wins, 90862 losses (winRatio=0.100570, priRatio=0.100000)
    data_test.go:976: 0.100000 vs 1.000000: 9184 wins, 90816 losses (winRatio=0.101128, priRatio=0.100000)
    data_test.go:976: 0.100000 vs 2.000000: 4689 wins, 95311 losses (winRatio=0.049197, priRatio=0.050000)
    data_test.go:976: 0.100000 vs 10.000000: 984 wins, 99016 losses (winRatio=0.009938, priRatio=0.010000)
    data_test.go:976: 0.500000 vs 1.000000: 33340 wins, 66660 losses (winRatio=0.500150, priRatio=0.500000)
    data_test.go:976: 0.500000 vs 1.000000: 33224 wins, 66776 losses (winRatio=0.497544, priRatio=0.500000)
    data_test.go:976: 0.500000 vs 2.000000: 19919 wins, 80081 losses (winRatio=0.248736, priRatio=0.250000)
    data_test.go:976: 0.500000 vs 10.000000: 4706 wins, 95294 losses (winRatio=0.049384, priRatio=0.050000)
    data_test.go:976: 0.500000 vs 100.000000: 481 wins, 99519 losses (winRatio=0.004833, priRatio=0.005000)
    data_test.go:976: 1.000000 vs 1.000000: 49880 wins, 50120 losses (winRatio=0.995211, priRatio=1.000000)
    data_test.go:976: 1.000000 vs 2.000000: 33218 wins, 66782 losses (winRatio=0.497409, priRatio=0.500000)
    data_test.go:976: 1.000000 vs 10.000000: 9119 wins, 90881 losses (winRatio=0.100340, priRatio=0.100000)
    data_test.go:976: 1.000000 vs 100.000000: 988 wins, 99012 losses (winRatio=0.009979, priRatio=0.010000)
    data_test.go:976: 1.000000 vs 999.000000: 80 wins, 99920 losses (winRatio=0.000801, priRatio=0.001001)
    data_test.go:979: Error (1.000000 vs 999.000000: 80 wins, 99920 losses): winRatio=0.000801 not close to priRatio=0.001001
    data_test.go:976: 1.000000 vs 2.000000: 33289 wins, 66711 losses (winRatio=0.499003, priRatio=0.500000)
    data_test.go:976: 1.000000 vs 10.000000: 9169 wins, 90831 losses (winRatio=0.100946, priRatio=0.100000)
    data_test.go:976: 1.000000 vs 100.000000: 984 wins, 99016 losses (winRatio=0.009938, priRatio=0.010000)
    data_test.go:976: 1.000000 vs 999.000000: 92 wins, 99908 losses (winRatio=0.000921, priRatio=0.001001)
    data_test.go:976: 2.000000 vs 10.000000: 16684 wins, 83316 losses (winRatio=0.200250, priRatio=0.200000)
    data_test.go:976: 2.000000 vs 100.000000: 1927 wins, 98073 losses (winRatio=0.019649, priRatio=0.020000)
    data_test.go:976: 2.000000 vs 999.000000: 167 wins, 99833 losses (winRatio=0.001673, priRatio=0.002002)
    data_test.go:976: 10.000000 vs 100.000000: 9173 wins, 90827 losses (winRatio=0.100994, priRatio=0.100000)
    data_test.go:976: 10.000000 vs 999.000000: 973 wins, 99027 losses (winRatio=0.009826, priRatio=0.010010)
    data_test.go:976: 100.000000 vs 999.000000: 9170 wins, 90830 losses (winRatio=0.100958, priRatio=0.100100)
--- FAIL: TestMakePriority (0.03s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
, <code>stress=true</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMakePriority.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachpb.TestMakePriority [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/11724270?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/11724270?buildTab=artifacts#/) on master @ [d02071779bdfd01136b71ff720bdcf0e3d0c8c62](https://github.com/cockroachdb/cockroach/commits/d02071779bdfd01136b71ff720bdcf0e3d0c8c62):


```
=== RUN   TestMakePriority
    data_test.go:977: 0.001100 vs 0.010000: 10047 wins, 89953 losses (winRatio=0.111692, priRatio=0.110000)
    data_test.go:977: 0.001100 vs 0.100000: 1066 wins, 98934 losses (winRatio=0.010775, priRatio=0.011000)
    data_test.go:977: 0.001100 vs 0.500000: 230 wins, 99770 losses (winRatio=0.002305, priRatio=0.002200)
    data_test.go:977: 0.001100 vs 1.000000: 143 wins, 99857 losses (winRatio=0.001432, priRatio=0.001100)
    data_test.go:980: Error (0.001100 vs 1.000000: 143 wins, 99857 losses): winRatio=0.001432 not close to priRatio=0.001100
    data_test.go:977: 0.001100 vs 1.000000: 123 wins, 99877 losses (winRatio=0.001232, priRatio=0.001100)
    data_test.go:977: 0.010000 vs 0.100000: 9048 wins, 90952 losses (winRatio=0.099481, priRatio=0.100000)
    data_test.go:977: 0.010000 vs 0.500000: 2082 wins, 97918 losses (winRatio=0.021263, priRatio=0.020000)
    data_test.go:977: 0.010000 vs 1.000000: 1000 wins, 99000 losses (winRatio=0.010101, priRatio=0.010000)
    data_test.go:977: 0.010000 vs 1.000000: 1025 wins, 98975 losses (winRatio=0.010356, priRatio=0.010000)
    data_test.go:977: 0.010000 vs 2.000000: 495 wins, 99505 losses (winRatio=0.004975, priRatio=0.005000)
    data_test.go:977: 0.100000 vs 0.500000: 16702 wins, 83298 losses (winRatio=0.200509, priRatio=0.200000)
    data_test.go:977: 0.100000 vs 1.000000: 9028 wins, 90972 losses (winRatio=0.099239, priRatio=0.100000)
    data_test.go:977: 0.100000 vs 1.000000: 9005 wins, 90995 losses (winRatio=0.098961, priRatio=0.100000)
    data_test.go:977: 0.100000 vs 2.000000: 4833 wins, 95167 losses (winRatio=0.050784, priRatio=0.050000)
    data_test.go:977: 0.100000 vs 10.000000: 998 wins, 99002 losses (winRatio=0.010081, priRatio=0.010000)
    data_test.go:977: 0.500000 vs 1.000000: 33347 wins, 66653 losses (winRatio=0.500308, priRatio=0.500000)
    data_test.go:977: 0.500000 vs 1.000000: 33226 wins, 66774 losses (winRatio=0.497589, priRatio=0.500000)
    data_test.go:977: 0.500000 vs 2.000000: 20223 wins, 79777 losses (winRatio=0.253494, priRatio=0.250000)
    data_test.go:977: 0.500000 vs 10.000000: 4883 wins, 95117 losses (winRatio=0.051337, priRatio=0.050000)
    data_test.go:977: 0.500000 vs 100.000000: 543 wins, 99457 losses (winRatio=0.005460, priRatio=0.005000)
    data_test.go:977: 1.000000 vs 1.000000: 49967 wins, 50033 losses (winRatio=0.998681, priRatio=1.000000)
    data_test.go:977: 1.000000 vs 2.000000: 33623 wins, 66377 losses (winRatio=0.506546, priRatio=0.500000)
    data_test.go:977: 1.000000 vs 10.000000: 9142 wins, 90858 losses (winRatio=0.100619, priRatio=0.100000)
    data_test.go:977: 1.000000 vs 100.000000: 1011 wins, 98989 losses (winRatio=0.010213, priRatio=0.010000)
    data_test.go:977: 1.000000 vs 999.000000: 95 wins, 99905 losses (winRatio=0.000951, priRatio=0.001001)
    data_test.go:977: 1.000000 vs 2.000000: 33484 wins, 66516 losses (winRatio=0.503398, priRatio=0.500000)
    data_test.go:977: 1.000000 vs 10.000000: 9090 wins, 90910 losses (winRatio=0.099989, priRatio=0.100000)
    data_test.go:977: 1.000000 vs 100.000000: 987 wins, 99013 losses (winRatio=0.009968, priRatio=0.010000)
    data_test.go:977: 1.000000 vs 999.000000: 92 wins, 99908 losses (winRatio=0.000921, priRatio=0.001001)
    data_test.go:977: 2.000000 vs 10.000000: 16641 wins, 83359 losses (winRatio=0.199631, priRatio=0.200000)
    data_test.go:977: 2.000000 vs 100.000000: 2027 wins, 97973 losses (winRatio=0.020689, priRatio=0.020000)
    data_test.go:977: 2.000000 vs 999.000000: 192 wins, 99808 losses (winRatio=0.001924, priRatio=0.002002)
    data_test.go:977: 10.000000 vs 100.000000: 8959 wins, 91041 losses (winRatio=0.098406, priRatio=0.100000)
    data_test.go:977: 10.000000 vs 999.000000: 961 wins, 99039 losses (winRatio=0.009703, priRatio=0.010010)
    data_test.go:977: 100.000000 vs 999.000000: 9209 wins, 90791 losses (winRatio=0.101431, priRatio=0.100100)
--- FAIL: TestMakePriority (0.04s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMakePriority.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachpb.TestMakePriority [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/11726441?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/11726441?buildTab=artifacts#/) on master @ [2810022ece57c99bfcaab3d5bb86206a7128f144](https://github.com/cockroachdb/cockroach/commits/2810022ece57c99bfcaab3d5bb86206a7128f144):


```
=== RUN   TestMakePriority
F230911 17:59:21.454608 96 roachpb/data.go:1231  [-] 3  updating txn meta={id=e6093bd2 key=/Min iso=Serializable pri=0.00000000 epo=0 ts=0,0 min=0,0 seq=0} lock=false stat=PENDING rts=0,0 wto=false gul=0,0 with different txn "name" meta={id=d228bbb4 key="foo" iso=Snapshot pri=44.58039917 epo=2 ts=0.000000020,21? min=0.000000010,11? seq=123} lock=true stat=COMMITTED rts=0.000000020,22? wto=true gul=0.000000040,41? int=1 ifw=1 isn=1
F230911 17:59:21.454608 96 roachpb/data.go:1231  [-] 3 !stack trace omitted via SetExitFunc()
    data_test.go:977: 0.001100 vs 0.010000: 9945 wins, 90055 losses (winRatio=0.110433, priRatio=0.110000)
    data_test.go:977: 0.001100 vs 0.100000: 1086 wins, 98914 losses (winRatio=0.010979, priRatio=0.011000)
    data_test.go:977: 0.001100 vs 0.500000: 229 wins, 99771 losses (winRatio=0.002295, priRatio=0.002200)
    data_test.go:977: 0.001100 vs 1.000000: 102 wins, 99898 losses (winRatio=0.001021, priRatio=0.001100)
    data_test.go:977: 0.001100 vs 1.000000: 116 wins, 99884 losses (winRatio=0.001161, priRatio=0.001100)
    data_test.go:977: 0.010000 vs 0.100000: 9254 wins, 90746 losses (winRatio=0.101977, priRatio=0.100000)
    data_test.go:977: 0.010000 vs 0.500000: 1886 wins, 98114 losses (winRatio=0.019223, priRatio=0.020000)
    data_test.go:977: 0.010000 vs 1.000000: 1035 wins, 98965 losses (winRatio=0.010458, priRatio=0.010000)
    data_test.go:977: 0.010000 vs 1.000000: 1010 wins, 98990 losses (winRatio=0.010203, priRatio=0.010000)
    data_test.go:977: 0.010000 vs 2.000000: 483 wins, 99517 losses (winRatio=0.004853, priRatio=0.005000)
    data_test.go:977: 0.100000 vs 0.500000: 16685 wins, 83315 losses (winRatio=0.200264, priRatio=0.200000)
    data_test.go:977: 0.100000 vs 1.000000: 9068 wins, 90932 losses (winRatio=0.099723, priRatio=0.100000)
    data_test.go:977: 0.100000 vs 1.000000: 9021 wins, 90979 losses (winRatio=0.099155, priRatio=0.100000)
    data_test.go:977: 0.100000 vs 2.000000: 4660 wins, 95340 losses (winRatio=0.048878, priRatio=0.050000)
    data_test.go:977: 0.100000 vs 10.000000: 1036 wins, 98964 losses (winRatio=0.010468, priRatio=0.010000)
    data_test.go:977: 0.500000 vs 1.000000: 33122 wins, 66878 losses (winRatio=0.495260, priRatio=0.500000)
    data_test.go:977: 0.500000 vs 1.000000: 33021 wins, 66979 losses (winRatio=0.493005, priRatio=0.500000)
    data_test.go:977: 0.500000 vs 2.000000: 19780 wins, 80220 losses (winRatio=0.246572, priRatio=0.250000)
    data_test.go:977: 0.500000 vs 10.000000: 4896 wins, 95104 losses (winRatio=0.051480, priRatio=0.050000)
    data_test.go:977: 0.500000 vs 100.000000: 516 wins, 99484 losses (winRatio=0.005187, priRatio=0.005000)
    data_test.go:977: 1.000000 vs 1.000000: 49827 wins, 50173 losses (winRatio=0.993104, priRatio=1.000000)
    data_test.go:977: 1.000000 vs 2.000000: 33220 wins, 66780 losses (winRatio=0.497454, priRatio=0.500000)
    data_test.go:977: 1.000000 vs 10.000000: 9194 wins, 90806 losses (winRatio=0.101249, priRatio=0.100000)
    data_test.go:977: 1.000000 vs 100.000000: 1022 wins, 98978 losses (winRatio=0.010326, priRatio=0.010000)
    data_test.go:977: 1.000000 vs 999.000000: 119 wins, 99881 losses (winRatio=0.001191, priRatio=0.001001)
    data_test.go:977: 1.000000 vs 2.000000: 33609 wins, 66391 losses (winRatio=0.506228, priRatio=0.500000)
    data_test.go:977: 1.000000 vs 10.000000: 9380 wins, 90620 losses (winRatio=0.103509, priRatio=0.100000)
    data_test.go:977: 1.000000 vs 100.000000: 1071 wins, 98929 losses (winRatio=0.010826, priRatio=0.010000)
    data_test.go:977: 1.000000 vs 999.000000: 131 wins, 99869 losses (winRatio=0.001312, priRatio=0.001001)
    data_test.go:980: Error (1.000000 vs 999.000000: 131 wins, 99869 losses): winRatio=0.001312 not close to priRatio=0.001001
    data_test.go:977: 2.000000 vs 10.000000: 16885 wins, 83115 losses (winRatio=0.203152, priRatio=0.200000)
    data_test.go:977: 2.000000 vs 100.000000: 2040 wins, 97960 losses (winRatio=0.020825, priRatio=0.020000)
    data_test.go:977: 2.000000 vs 999.000000: 234 wins, 99766 losses (winRatio=0.002345, priRatio=0.002002)
    data_test.go:977: 10.000000 vs 100.000000: 9225 wins, 90775 losses (winRatio=0.101625, priRatio=0.100000)
    data_test.go:977: 10.000000 vs 999.000000: 1028 wins, 98972 losses (winRatio=0.010387, priRatio=0.010010)
    data_test.go:977: 100.000000 vs 999.000000: 9164 wins, 90836 losses (winRatio=0.100885, priRatio=0.100100)
--- FAIL: TestMakePriority (0.04s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMakePriority.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/roachpb/BUILD.bazel
+++ b/pkg/roachpb/BUILD.bazel
@@ -77,7 +77,6 @@ go_test(
         "//pkg/kv/kvserver/concurrency/isolation",
         "//pkg/kv/kvserver/concurrency/lock",
         "//pkg/storage/enginepb",
-        "//pkg/testutils/skip",
         "//pkg/testutils/zerofields",
         "//pkg/util",
         "//pkg/util/bitarray",

--- a/pkg/roachpb/data_test.go
+++ b/pkg/roachpb/data_test.go
@@ -26,7 +26,6 @@ import (
 	"github.com/cockroachdb/cockroach/pkg/kv/kvserver/concurrency/isolation"
 	"github.com/cockroachdb/cockroach/pkg/kv/kvserver/concurrency/lock"
 	"github.com/cockroachdb/cockroach/pkg/storage/enginepb"
-	"github.com/cockroachdb/cockroach/pkg/testutils/skip"
 	"github.com/cockroachdb/cockroach/pkg/testutils/zerofields"
 	"github.com/cockroachdb/cockroach/pkg/util"
 	"github.com/cockroachdb/cockroach/pkg/util/bitarray"
@@ -909,7 +908,6 @@ func checkVal(val, expected, errFraction float64) bool {
 // in MakePriority returning priorities that are P times more likely
 // to be higher than a priority with user priority = 1.
 func TestMakePriority(t *testing.T) {
-	skip.WithIssue(t, 110303, "flaky test")
 	// Verify min & max.
 	if a, e := MakePriority(MinUserPriority), enginepb.MinTxnPriority; a != e {
 		t.Errorf("expected min txn priority %d; got %d", e, a)
@@ -932,7 +930,7 @@ func TestMakePriority(t *testing.T) {
 	}
 
 	// Generate values for all priorities.
-	const trials = 100000
+	const trials = 750000
 	values := make([][trials]enginepb.TxnPriority, len(userPs))
 	for i, userPri := range userPs {
 		for tr := 0; tr < trials; tr++ {

